E2_ESRGAN/lib/utils.py

```diff
@@ -164,19 +164,24 @@ class RDB(tf.keras.layers.Layer):
 
   def __init__(self, out_features=32, bias=True):
     super(RDB, self).__init__()
-    self.conv = lambda x: tf.keras.layers.Conv2D(
+    conv= lambda: tf.keras.layers.Conv2D(
         out_features,
         kernel_size=[3, 3],
-        strides=[1, 1], padding="same", use_bias=bias)(x)
+        strides=[1, 1], padding="same", use_bias=bias)
+    self.conv_1 = conv()
```

Comment:
Can you keep these in a list and use list index instead of number in variable name?

Reply:
Even I was thinking so, it would be really easy. but in most of the models they do this.